content/_index.md

```diff
@@ -9,6 +9,10 @@ slideshow:
   photo: "/img/holiday-lights-home-slider.jpg"
   title_mobile: Holiday Lights Nov 28 - Dec 31
   photo_mobile: "/img/holiday-lights-mobile.jpg"
+  video:
+    enabled: true
+    source: "http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4"
+    type: "mp4"
```

Comment:
What is the `enabled` bool for? I was thinking if the user adds a source, which they will upload through CMS (or else) then we have a video. We should always keep the Front Matter to a minimum, but I might be missing something.
Also, we will enforce one type for the client to use. So let's assume it's always `.mp4`